beagle/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "1.69.1"
+__version__ = "1.70.0"
```

Comment:
You will probably need to update the version so its at or above this release: https://github.com/mskcc/beagle/pull/1172